akka-docs/src/main/paradox/common/circuitbreaker.md

```diff
@@ -92,33 +92,31 @@ Java
 
 @@@ note
 
-Using the `CircuitBreaker`'s companion object @scala[*apply*]@java[*create*] method
-will return a `CircuitBreaker` where callbacks are executed in the caller's thread.
-This can be useful if the asynchronous `Future` behavior is unnecessary, for
+Using the @scala[@apidoc[CircuitBreaker](CircuitBreaker$)'s companion object @scaladoc[apply](akka.pattern.CircuitBreaker$#apply(scheduler:akka.actor.Scheduler,maxFailures:Int,callTimeout:scala.concurrent.duration.FiniteDuration,resetTimeout:scala.concurrent.duration.FiniteDuration):akka.pattern.CircuitBreaker)]@java[@javadoc[CircuitBreaker.create](akka.pattern.CircuitBreaker#create(akka.actor.Scheduler,int,java.time.Duration,java.time.Duration))] method
+will return a @apidoc[CircuitBreaker] where callbacks are executed in the caller's thread.
+This can be useful if the asynchronous @scala[@scaladoc[Future](scala.concurrent.Future)]@java[@javadoc[CompletionState](java.util.concurrent.CompletionStage)] behavior is unnecessary, for
 example invoking a synchronous-only API.
 
 @@@
 
 ### Control failure count explicitly
 
-By default, the circuit breaker treats `Exception` as failure in synchronized API, or failed `Future` as failure in future based API.
+By default, the circuit breaker treats @javadoc[Exception](java.lang.Exception) as failure in synchronized API, or failed @scala[@scaladoc[Future](scala.concurrent.Future)]@java[@javadoc[CompletionState](java.util.concurrent.CompletionStage)] as failure in future based API.
 On failure, the failure count will increment. If the failure count reaches the *maxFailures*, the circuit breaker will be opened.
 However, some applications may require certain exceptions to not increase the failure count.
 In other cases one may want to increase the failure count even if the call succeeded.
 Akka circuit breaker provides a way to achieve such use cases:
 
-* `withCircuitBreaker`
-* `withSyncCircuitBreaker`
-* `callWithCircuitBreaker`
-* `callWithCircuitBreakerCS`
-* `callWithSyncCircuitBreaker`
+@scala[@scaladoc[withCircuitBreaker](akka.pattern.CircuitBreaker#withCircuitBreaker[T](body:=%3Escala.concurrent.Future[T],defineFailureFn:scala.util.Try[T]=%3EBoolean):scala.concurrent.Future[T])]@java[@javadoc[callWithCircuitBreaker](akka.pattern.CircuitBreaker#callWithCircuitBreaker(java.util.concurrent.Callable,java.util.function.BiFunction))]  
+@scala[@scaladoc[withSyncCircuitBreaker](akka.pattern.CircuitBreaker#withSyncCircuitBreaker[T](body:=%3ET,defineFailureFn:scala.util.Try[T]=%3EBoolean):T)]@java[@javadoc[callWithSyncCircuitBreaker](akka.pattern.CircuitBreaker#callWithSyncCircuitBreaker(java.util.concurrent.Callable,java.util.function.BiFunction))]  
```

Comment:
little trick. I can't use markdown "*", because of 2 methods in scala and 3 methods in java (one unnecessary "*"). Lines 110 and 111 contain two spaces in the end (it's a markdown new line).
Any different approach will be appreciated :)

Reply:
Isn't it possible to use blocks? Something like

```
@scala[* @scaladoc[]()
* @scaladoc[]()
]@java[* @scaladoc[]()
* @scaladoc[]()
* * @scaladoc[]()]
```

Reply:
Unfortunately not. It will transform to a single line like that:

<img width="867" alt="Снимок экрана 2021-09-24 в 23 59 42" src="https://user-images.githubusercontent.com/4740207/134739144-2a03e0f1-2954-4606-b54f-122603ad4668.png">

There are [some additional options in markdown](https://www.markdownguide.org/basic-syntax/#line-break-best-practices), but I'm not sure. Or it can be like a single line without "start"

Reply:
an alternative is to use commas and put them all on the same line. 

Java:
Akka circuit breaker provides a way to achieve such use cases: callWithCircuitBreaker, callWithSyncCircuitBreaker and callWithCircuitBreakerCS

Scala: 
Akka circuit breaker provides a way to achieve such use cases: withCircuitBreaker and withSyncCircuitBreaker

Reply:
something like:

```
Akka circuit breaker provides a way to achieve such use cases: @scala[@scaladoc[withCircuitBreaker](akka.pattern.CircuitBreaker#withCircuitBreaker[T](body:=%3Escala.concurrent.Future[T],defineFailureFn:scala.util.Try[T]=%3EBoolean):scala.concurrent.Future[T])]@java[@javadoc[callWithCircuitBreaker](akka.pattern.CircuitBreaker#callWithCircuitBreaker(java.util.concurrent.Callable,java.util.function.BiFunction))]@scala[ and @scaladoc[withSyncCircuitBreaker](akka.pattern.CircuitBreaker#withSyncCircuitBreaker[T](body:=%3ET,defineFailureFn:scala.util.Try[T]=%3EBoolean):T)]@java[, @javadoc[callWithSyncCircuitBreaker](akka.pattern.CircuitBreaker#callWithSyncCircuitBreaker(java.util.concurrent.Callable,java.util.function.BiFunction))] @java[and @javadoc[callWithCircuitBreakerCS](akka.pattern.CircuitBreaker#callWithCircuitBreakerCS(java.util.concurrent.Callable,java.util.function.BiFunction))]
```

(sorry, I can't make it a GH suggestion because the first line is not included in the diff.)